cinema/models.py

```diff
@@ -35,12 +38,18 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> str:
+    ext = filename.split(".")[-1]
+    return f"upload/movies/{slugify(instance.title)}-{uuid.uuid4()}.{ext}"
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(null=True, upload_to=movie_image_path)
```

Comment:
Ensure that the `ImageField` in the `Movie` model is not accessible via POST requests to `api/cinema/movies/`, as per the task requirements. This field should only be used via the `/upload-image/` endpoint.